ale/drivers/mex_drivers.py

```diff
@@ -69,7 +69,7 @@
             [-8569.5561557859, 0.068566503695773, 142.857126402363]],
 }
 
-class MexHrscPds3NaifSpiceDriver(LineScanner, Pds3Label, NaifSpice, RadialDistortion, Driver):
+class MexHrscPds3NaifSpiceDriver(LineScanner, Pds3Label, NaifSpice, NoDistortion, Driver):
```

Comment:
why disable distortion?

Reply:
I think this was before we had a `NoDistortion` class/mixin in ALE. In the driver we define the distortion coefficients as all zeros:
```
    @property
    def odtk(self):
        """
        The coefficients for the distortion model
        Returns
        -------
        : list
          Radial distortion coefficients. There is only one coefficient for LROC NAC l/r
        """
        return [0.0, 0.0, 0.0]
```

I just changed it to remove duplicate code but didn't remove the odtk property. I will do that